test/unit/test-layout.cxx

```diff
@@ -85,7 +82,7 @@ TEST(LayoutTest, 2D_JI)
    * (-1, -1), (0, -1), (1, -1)
    * (-1, -2), (0, -2), (1, -2)
    */
-  const my_layout layout({-1,-2}, {1,0});
+  const my_layout layout = my_layout::from_layout({-1,-2}, RAJA::make_permuted_layout<RAJA::PERM_JI>(3,3));
```

Comment:
That would certainly be easy enough to add, I'll throw one together momentarily.